scripts/prove_stdio.sh

```diff
@@ -97,6 +97,7 @@ fi
 if [[ $TEST_ONLY == "test_only" ]]; then
     cargo run --quiet --release --package zero --bin leader -- \
       --test-only \
+      --use-test-config \
```

Comment:
same comments as in `prove_rpc.sh` for here and below

Reply:
Done.